### DISCUSSION TITLE
Adding Amiga? (SillyTavern and EmulatorJS)

### DISCUSSION THREAD

**Discussion by TheRealMiRRoRMaN** [EmulatorJS/EmulatorJS#866] [Q&A]
Hi,

I see that its also possible to play Amiga Games in this, however I haven't been able to find where? It does have an option for Commodore 64 and Playstation which all work fine for me, but i'd really love to be able to also play Amiga games in this.

I searched high and low for the answers but I wasn't able to find them. I'm hoping someone can help me out, I have all the BIOS roms etc

**Comment by ghost** [comment]
Trying to play my amiga + amiga 1200 games and wondering if this Issue works. Reviewing Docs and found this before hand.
